web/modules/custom/helloworld/src/Form/helloworldSettingsForm.php

```diff
@@ -0,0 +1,59 @@
+<?php 
+
+namespace Drupal\helloworld\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+class helloworldSettingsForm extends ConfigFormBase {
```

Comment:
class name should always begin with a capital letter.
Same for the class file name.

Reply:
Done.